### PR TITLE
3 - Repository prep structure

### PR DESCRIPTION
**Pull request by cdcabrera**
Baseline commit in preparation for a documentation/locale level repository.
* git and editor configuration
* NPM resources
* base ReadMe

Relates to research issue https://github.com/cloudigrade/frontigrade/issues/25

updates #3

### CODE REVIEW

File: README.md
Comment (elyezer) [inline README.md]: s/Cloud Meter/Cloudigrade?

**Review by elyezer — DISMISSED** [review]
LGTM, left a small comment that you may want to handle before merging.

File: README.md
Comment (cdcabrera) [reply README.md]: Can update it since its the org name +1

**Review by elyezer — APPROVED** [review]

**Comment by elyezer** [comment]
I am approving but I am not 100% sure if we should have Cloud Meter on the README, but anyway having Cloudigrade will make people aware of what this is related to

**Comment by cdcabrera** [comment]
@elyezer gotcha, we use some of the Cloud Meter labeling inside Frontigrade. But agreed, we should probably bring it forward and look at cleaning it up

**Pull request merged**
